Sources/MongoSwift/ReadWriteConcern.swift

```diff
@@ -170,27 +166,40 @@ public class WriteConcern: Codable {
     }
 
     /// Indicates the `W` value for this `WriteConcern`.
-    public var w: W {
-        if let wTag = mongoc_write_concern_get_wtag(self._writeConcern) {
-            return .tag(String(cString: wTag))
-        }
+    public var w: W? {
         let number = mongoc_write_concern_get_w(self._writeConcern)
-        if number == MONGOC_WRITE_CONCERN_W_MAJORITY { return .majority }
-        return .number(number)
+        switch number {
+        case MONGOC_WRITE_CONCERN_W_DEFAULT:
+            return nil
+        case MONGOC_WRITE_CONCERN_W_MAJORITY:
+            return .majority
+        case MONGOC_WRITE_CONCERN_W_TAG:
+            if let wTag = mongoc_write_concern_get_wtag(self._writeConcern) {
+                return .tag(String(cString: wTag))
+            }
+            fallthrough
```

Comment:
if for some reason this is `MONGOC_WRITE_CONCERN_W_TAG` but we don't get a `wtag` back, just fall back to returning `.number` with the number